shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -275,7 +277,7 @@ class DiffTests extends org.scalatest.funsuite.AnyFunSuite {
               }
             )
 
-            var results: (Str, Bool) \/ Opt[Ls[Ls[Str]]] = if (!allowTypeErrors &&
+            var results: (Str, Bool) \/ Opt[Ls[Str \/ Str]] = if (!allowTypeErrors &&
```

Comment:
`Str \/ Str` could just be a `(Bool, Str)`

Reply:
I see.